src/Client/src/app/admin-panel/employees/add-supervisor/add-supervisor.component.html

```diff
@@ -0,0 +1,69 @@
+<app-loading-spinner *ngIf="loading"></app-loading-spinner>
+<form [formGroup]="form" (ngSubmit)="onSubmit()" *ngIf="!loading">
+  <h2 mat-dialog-title>Add supervisor</h2>
+  <mat-dialog-content class="mat-typography">
+    <div class="row">
+        <div class="mb-3 col-6 offset-3">
+          <label class="form-label" for="username">Username</label>
+          <input formControlName="username" type="text" id="username" class="form-control" />
+          <span *ngIf="form.get('username').invalid && form.get('username').touched" class="form-text text-danger mb-2">
+            <span *ngIf="form.get('username').hasError('required')">
+              Username is required!
+            </span>
+            <span *ngIf="form.get('username').hasError('minlength')">
+                Username must have minimum 2 character!
+            </span>
+            <span *ngIf="form.get('username').hasError('maxlength')">
+                Username must have maximum 50 characters!
+            </span>
+          </span>
+        </div>
+  
+        <div class="mb-3 col-6 offset-3">
+          <label class="form-label" for="email">Email</label>
+          <input formControlName="email" type="text" id="email" class="form-control" />
+          <span *ngIf="form.get('email').invalid && form.get('email').touched" class="form-text text-danger mb-2">
+            <span *ngIf="form.get('email').hasError('required')">
+                Email is required!
+            </span>
+            <span *ngIf="form.get('email').hasError('minlength')">
+                Email must have minimum 2 character!
+            </span>
+            <span *ngIf="form.get('email').hasError('maxlength')">
+                Email must have maximum 50 characters!
+            </span>
+            <span *ngIf="form.get('email').hasError('email')">
+              Email is invalid!
+          </span>
+          </span>
+        </div>
+  
+        <div class="mb-3 col-6 offset-3">
+          <label class="form-label" for="password">Password</label>
+          <input formControlName="password" type="password" id="password" class="form-control" />
+          <span *ngIf="form.get('password').invalid && form.get('password').touched" class="form-text text-danger mb-2">
+            <span *ngIf="form.get('password').hasError('required')">
+                Password is required!
+            </span>
+            <span *ngIf="form.get('password').hasError('minlength')">
+                Password must have minimum 2 character!
```

Comment:
Nie pokrywa się to z walidacją hasła, którą mamy przy rejestracji. W związku z tym mogę stworzyć supervisora, którego hasło ma mniej niż 8 znaków, ale przez to nie mogę się zalogować na jego konto.

Reply:
Fixed